gemini/sample-apps/finance-advisor-spanner/pages/1_Asset_Search.py

```diff
@@ -0,0 +1,134 @@
+"""This module is the page for Asset Search feature"""
+
+# pylint: disable=line-too-long, invalid-name, import-error, use-dict-literal, duplicate-code
+
+import time as t
+
+from css import favicon, footer
+from database import display_spanner_query, fts_query, like_query
+from itables.streamlit import interactive_table
+import streamlit as st
+
+st.set_page_config(
+    layout="wide",
+    page_title="FinVest Advisor",
+    page_icon=favicon,
+    initial_sidebar_state="expanded",
+)
+st.logo(
+    "https://storage.googleapis.com/github-repo/generative-ai/sample-apps/finance-advisor-spanner/images/investments.png"
+)
+
+
+def local_css(file_name: str) -> None:
+    """This loads local css"""
+    with open(file_name, encoding="utf-8") as f:
+        st.markdown(f"<style>{f.read()}</style>", unsafe_allow_html=True)
+
+
+local_css("pages/styles.css")
+
+
+def asset_search_precise() -> None:
+    """This function implements Asset search LIKE Query"""
+
+    st.header("FinVest Fund Advisor")
+    st.subheader("Asset Search")
+
+    st.columns([0.25, 0.25, 0.20, 0.10])
+    classes = ["display", "compact", "cell-border", "stripe"]
+    buttons = ["pageLength", "csvHtml5", "excelHtml5", "colvis"]
+    style = "table-layout:auto;width:auto;margin:auto;caption-side:bottom"
+    it_args = dict(
+        classes=classes,
+        style=style,
+    )
+
+    if buttons:
+        it_args["buttons"] = buttons
+
+    query_params = []
+    query_params.append(investment_strategy_pt1.strip())
+    query_params.append(and_or_exclude)
+    query_params.append(investment_strategy_pt2.strip())
+    query_params.append(investment_manager.strip())
+
+    with st.spinner("Querying Spanner..."):
+        t.sleep(1)
+
+        return_vals = like_query(query_params)
+        spanner_query = return_vals.get("query")
+        data = return_vals.get("data")
+        display_spanner_query(str(spanner_query))
+
+    interactive_table(data, caption="", **it_args)
+
+
+def asset_search() -> None:
+    """This function immplements Asset Search"""
```

Comment:
```suggestion
    """This function implements Asset Search"""
```
Fix other spelling errors throughout